src/stream.c

```diff
@@ -756,3 +756,34 @@ struct call *stream_call(const struct stream *strm)
 {
 	return strm ? strm->call : NULL;
 }
+
+/**
+ * Get the sdp object from the stream
+ *
+ * @param strm Stream object
+ *
+ * @return SDP media object
+ */
+const struct sdp_media *stream_sdp(const struct stream *strm)
+{
+	return strm ? strm->sdp : NULL;
+}
+
+uint32_t metric_get_tx_n_packets(const struct stream *strm) {
+	return strm ? strm->metric_tx.n_packets : 0;
+}
```

Comment:
please move the starting curly bracket to the beginning of the next line, so that it looks like this:

```
void foo(void)
{
}
```

the same applies for all the other added functions below